.github/workflows/publish-qemu-bin.yml

```diff
@@ -0,0 +1,163 @@
+        QEMU_VERSION: ${{ steps.qemu-version.outputs.QEMU_VERSION }}
+      run: |
+        choco install qemu --version=$env:QEMU_VERSION
+    - name: Stage Qemu Binaries
+      run: |
+        mkdir temp
+        mkdir temp\share
+        cp "C:\Program Files\qemu\qemu-system-x86_64.exe" temp
+        cp "C:\Program Files\qemu\qemu-system-aarch64.exe" temp
+        cp "C:\Program Files\qemu\share\kvmvapic.bin" temp\share
+        cp "C:\Program Files\qemu\share\vgabios-cirrus.bin" temp\share
+        cp "C:\Program Files\qemu\share\vgabios-stdvga.bin" temp\share
+        Get-ChildItem -Path "C:\Program Files\qemu" -Filter *.dll | Move-Item -Destination "temp"
+        
+    - name: Upload Qemu Artifact
+      uses: actions/upload-artifact@v4
+      with:
+        name: qemu-windows
+        path: |
+          temp\*
+        if-no-files-found: error
+
+  qemu-linux:
+    name: Build Qemu for Linux
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+    - name: Get current Qemu version
+      id: qemu-version
+      run: |
+        qemuVersion=$(jq -r .linux $QEMU_VERSION_FILE)
+        echo "QEMU_VERSION=$qemuVersion" >> $GITHUB_OUTPUT
+      shell: bash
+    - name: Install Dependencies
+      run: |
+        sudo apt-get update
+        sudo apt-get install --yes --no-install-recommends \
+          autoconf \
+          automake \
+          autotools-dev \
+          bc \
+          build-essential \
+          dosfstools \
+          gcc \
+          libglib2.0-dev \
+          libgtk-3-dev \
+          libpixman-1-dev \
+          libsdl2-dev \
+          mtools \
+          ninja-build \
+          tar
+    - name: Compile Qemu ${{ steps.qemu-version.outputs.QEMU_VERSION }}
+      env:
+        QEMU_VERSION: ${{ steps.qemu-version.outputs.QEMU_VERSION }}
+        QEMU_URL: "https://gitlab.com/qemu-project/qemu.git"
+        DEBIAN_FRONTEND: "noninteractive"
+      run: |
+        git clone $QEMU_URL --branch $QEMU_VERSION --depth 1 qemu
+        cd qemu
+        mkdir bin
+        cd bin
+        ./../configure --target-list=x86_64-softmmu,aarch64-softmmu --enable-gtk
+        sudo make -j $(nproc)
+    - name: Stage Qemu Binaries
+      run: |
+        mkdir -p temp/share
+        cp qemu/bin/qemu-system-x86_64 qemu/bin/qemu-system-aarch64 temp
+        cp qemu/bin/qemu-bundle/usr/local/share/qemu/vgabios-cirrus.bin temp/share
+        cp qemu/bin/qemu-bundle/usr/local/share/qemu/vgabios-stdvga.bin temp/share
+        cp qemu/bin/qemu-bundle/usr/local/share/qemu/kvmvapic.bin temp/share
+    - name: Upload Qemu Artifact
+      uses: actions/upload-artifact@v4
+      with:
+        name: qemu-linux
+        path: |
+          temp/*
+        if-no-files-found: error
+  
+  prepare-extdep:
+    name: Prepare and Upload Qemu External Dependency
+    needs: [qemu-windows, qemu-linux]
+    runs-on: Ubuntu-latest
+    permissions:
+      contents: write
+
+    steps:
+      - name: Checkout

```

Comment:
Doesnt seem this checkout is actually used.

Reply:
Removed, thanks!

Reply:
Just kidding. Had to add it back in - but now it's being used.